client/src/App.js

```diff
@@ -16,7 +16,7 @@ import {
   isSMobile,
   isMobile,
   isTablet,
-  isDesktop,
```

Comment:
Lots of the changes on your files are removing the comma from the end of the object. I'm wondering if you're code editor is configured correctly to follow the eslint rules?